docs/recipes/build/add-build-script.md

```diff
@@ -0,0 +1,121 @@
+
+> The standard template comes [with a FAKE project](../../../template-safe-commands) by default, so **this recipe only applies to the minimal template**.
+
+---
+#### 1. Create a build project
+
+Create a new console app called 'Build' at the root of your solution
+
+```fsharp
+dotnet new console -lang f# -n Build -o .
+```
+
+> We are creating the project directly at the root of the solution in order to allow us to execute the build without needing to navigate into a subfolder.
+
+#### 2. Create a build script
+
+Open the project you just created in your IDE and rename the module it contains from `Program.fs` to `Build.fs`.
+
+This renaming isn't explicitly necessary, however it keeps your solution consistent with other SAFE apps and is a better name for the file really.
+
+> If you just rename the file directly rather than in your IDE, then the Build project won't be able to find it unless you edit the Build.fsproj file as well
+
+Open `Build.fs` and paste in the following code.
+
+```fsharp
+open Fake.Core
+open Fake.IO
+open System
+
+let redirect createProcess =
+    createProcess
+    |> CreateProcess.redirectOutputIfNotRedirected
+    |> CreateProcess.withOutputEvents Console.WriteLine Console.WriteLine
+
+let createProcess exe arg dir =
+    CreateProcess.fromRawCommandLine exe arg
+    |> CreateProcess.withWorkingDirectory dir
+    |> CreateProcess.ensureExitCode
+
+let dotnet = createProcess "dotnet"
+
+let npm =
+    let npmPath =
+        match ProcessUtils.tryFindFileOnPath "npm" with
+        | Some path -> path
+        | None -> failwith "npm was not found in path."
+    createProcess npmPath
+
+let run proc arg dir =
+    proc arg dir
+    |> Proc.run
+    |> ignore
+
+let execContext = Context.FakeExecutionContext.Create false "build.fsx" [ ]
+Context.setExecutionContext (Context.RuntimeContext.Fake execContext)
+
+Target.create "Clean" (fun _ -> Shell.cleanDir (Path.getFullName "deploy"))
+
+Target.create "InstallClient" (fun _ -> run npm "install" ".")
+
+Target.create "Run" (fun _ ->
+    run dotnet "build" (Path.getFullName "src/Shared")
+    [ dotnet "watch run" (Path.getFullName "src/Server")
+      dotnet "fable watch --run npx vite" (Path.getFullName "src/Client") ]
+    |> Seq.toArray
+    |> Array.map redirect
+    |> Array.Parallel.map Proc.run
+    |> ignore
+)
+
+open Fake.Core.TargetOperators
+
+let dependencies = [
+    "Clean"
+        ==> "InstallClient"
+        ==> "Run"
+]
+
+[<EntryPoint>]
+let main args =
+  try
+      match args with
+      | [| target |] -> Target.runOrDefault target
+      | _ -> Target.runOrDefault "Run"
+      0
+  with e ->
+      printfn "%A" e
+      1
+```
+
+#### 3. Add the project to the solution
+
+Run the following command
+
+```bash
+dotnet sln add Build.fsproj
+```
+#### 4. Installing dependencies
```

Comment:
We should probably include the paket commands for installing the dependencies in this section.

Reply:
Could be done in a separate PR.